notifier.rst

```diff
@@ -0,0 +1,562 @@
+.. index::
+   single: Notifier
+
+Sending Notifications with Notifier
```

Comment:
Do we need to include `Notifier` in the heading? We don't do that for other components. Maybe we can rename this as `Creating and Sending Notifications`.

Reply:
Looking from an SEO perspective, I would say it's nice to have the component's name in the main title of the page. But not sure if we need to do much SEO tricks on symfony.com